src/lib/viewers/controls/media/TimeControls.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { bdlBoxBlue, bdlGray62, white } from 'box-ui-elements/es/styles/variables';
+import SliderControl from '../slider';
+import './TimeControls.scss';
+
+export type Props = {
+    bufferedRange?: TimeRanges;
+    currentTime?: number;
+    durationTime?: number;
+    onTimeChange: (volume: number) => void;
+};
+
+export const round = (value: number): number => {
+    return +value.toFixed(4);
+};
+
+export const percent = (value1: number, value2: number): number => {
+    return round((value1 / value2) * 100);
+};
+
+export default function TimeControls({
+    bufferedRange,
+    currentTime = 0,
+    durationTime = 0,
+    onTimeChange,
+}: Props): JSX.Element {
+    const currentValue = percent(currentTime, durationTime);
+    const bufferedAmount = bufferedRange && bufferedRange.length ? bufferedRange.end(bufferedRange.length - 1) : 0;
+    const bufferedValue = percent(bufferedAmount, durationTime);
+
+    const handleChange = (newValue: number): void => {
+        onTimeChange(round(durationTime * (newValue / 100)));
+    };
+
+    return (
+        <SliderControl
+            className="bp-TimeControls"
+            onChange={handleChange}
+            step={0.1}
```

Comment:
It looks like we have this step property that is being passed in to SliderControl, but later in the SliderControl.tsx file, we're hardcoding step={1}. Does this prop still need to be passed down?

Reply:
Yes, as it overrides the default value of `1` in SliderControl.